### DISCUSSION TITLE
How to simply configure the git url in the .portfile of all ports as the mirror git repos of the company's internal ?

### DISCUSSION THREAD

**Discussion by nocanstillbb** [microsoft/vcpkg#35033] [General]
I have a port, and I want to merge it into vcpkg master, and also want to use it within the company.  but  company cannot access github(illegal), soI have to  push it to two remote manually,  and I don't want to maintain two `.portfile` 

is there a way to config some port's  git pull url when using vcpkg install with command , like `--overlay-ports` ,but don't need have two diffrent .protfile

**Comment by BillyONeal** [comment]
`porfile.cmake` is the source of truth for what a dependency is, so you would need to use overlay ports or another mechanism that replaces the port if you want the content to come from a different source.

Alternately, you can set up [Asset Caching](https://learn.microsoft.com/vcpkg/consume/asset-caching) for your business.

**Reply by nocanstillbb** [reply]
In the end, I resolved this issue by caching assets and binaries. I encountered significant setbacks while modifying the portfiles for Boost and OpenCV.

thank you
